src/pages/PublishedSchedule/PublishedSchedule.jsx

```diff
@@ -44,10 +51,12 @@ const PublishedSchedule = () => {
           return seasonOrder.indexOf(a.split(' ')[0]) - seasonOrder.indexOf(b.split(' ')[0]);
         }
       });
+
       setAllSeasons(data);
+
     };
-    renderTable();
-  }, []);
+    renderTable();    
+  }, [currentUser]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'curSeason'. Either include it or remove the dependency array.